test/navigation.vader

```diff
@@ -462,3 +465,27 @@ Execute (edit routes):
 
 Expect (generic template):
   <?php
+
+Before (blade file with __() function):
+  tabedit test/fixtures/laravel-8/resources/views/viewfile.blade.php
+
+Do (Move to (a)uth.failed, jump to translation file):
+  i{{ __('auth.failed') }}\<Esc>0fagf
+
+Execute:
+  AssertEqual expand('%'), 'resources/lang/auth.php'
```

Comment:
It's good to exercise the `gf` mapping at least once, as you have here, but it would also be valuable to test the output of `laravel#goto#filetype_php()` and `laravel#goto#filetype_blade()` directly. They output strings that are executed as navigation commands:

      AssertEqual laravel#goto#filetype_blade(), 'Elanguage en/auth'